systems/plants/@RigidBodyManipulator/addRobotFromSDF.m

```diff
@@ -312,6 +328,9 @@
         if (options.visual || options.visual_geometry)
          geometry = RigidBodyGeometry.parseSDFNode(geomnode,xyz,rpy,model,body.robotnum,options);
          if ~isempty(geometry)
+           if isa(geometry,'RigidBodyMesh')
+             geometry.scale=.0254; % gazebo dae meshes appear to be in inches
+           end
```

Comment:
I totally agree.  But what's the right solution here?  I don't see any information in the mesh that could be used to determine the units.

On Jan 8, 2015, at 3:06 PM, Scott Kuindersma <notifications@github.com> wrote:

> In systems/plants/@RigidBodyManipulator/addRobotFromSDF.m:
> 
> > @@ -312,6 +328,9 @@
> >          if (options.visual || options.visual_geometry)
> >           geometry = RigidBodyGeometry.parseSDFNode(geomnode,xyz,rpy,model,body.robotnum,options);
> >           if ~isempty(geometry)
> > +           if isa(geometry,'RigidBodyMesh')
> > +             geometry.scale=.0254; % gazebo dae meshes appear to be in inches
> > +           end
> Note: this could lead to confusion if someone tries loading a mesh that uses meters
> 
> —
> Reply to this email directly or view it on GitHub.
>